templates/ReactReduxSpa/ClientApp/boot-client.tsx

```diff
@@ -12,7 +12,7 @@ import * as RoutesModule from './routes';
 let routes = RoutesModule.routes;
 
 // Create browser history to use in the Redux store
-const baseUrl = document.getElementsByTagName('base')[0].getAttribute('href');
+const baseUrl = document.getElementsByTagName('base')[0].getAttribute('href')!;
```

Comment:
When we assume that a base element exists, we can also assume that it has the `href` attribute set.